src/inertia.js

```diff
@@ -42,6 +44,12 @@ export default {
     clearTimeout(this.progressBar)
   },
 
+  createToken() {
```

Comment:
I'm confused...I don't see a token actually created here. Where does the token actually come from?

Reply:
Line 48. Like I said, `token` isn’t a great name.

Reply:
Hah, sorry, not trying to be difficult, but on 48 you set the token to an empty object. At what point does the token actually have a value applied to it? I don't see that anywhere. It *seems* to me that token will only ever be an empty object. Help me understand. 😂

Reply:
Sorry @reinink! 😅 So the "token" _is_ the object. Objects are compared by **reference**, so `{} !== {}`. The key is that they are unique for each "visit."

Maybe "ID" is a better word: When we start a visit we assign a unique ID to that visit, and globally we keep track of the most recent visit ID. When a visit finally resolves (for example `inertia-vue` is about to render a new component) we check that the visit ID matches the most recent one. If it doesn’t it’s because a new visit was initiated since starting this one, so we want to skip over it.

Reply:
Ahh yes, that makes way more sense now. 😂 I'm obviously not a great JavaScript developer. Why am I making this framework again? 🤔